scripts/score_real_collection.py

```diff
@@ -27,14 +29,28 @@
         print(f"\tToken {collection.tokens[i]} has score: {token_score}")
 
     # Convert scores to rank
-    token_id_to_rank_score = extract_rank(
-        token_id_to_scores={
-            token_id: score for token_id, score in enumerate(token_scores)
-        }
-    )
+    token_id_to_scores = {
+        str(token_id): score for token_id, score in enumerate(token_scores)
+    }
 
     # Print out ranks and scores
-    for token_id, rank_score in token_id_to_rank_score.items():
+    print("Token ID and their ranks and scores, sorted by token ID")
```

Comment:
we should use logger instead.

Reply:
This is a one-off script meant for guidance on how to use library